presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -346,6 +232,8 @@ Extra Hidden Metadata Tables
 The Iceberg connector exposes extra hidden metadata tables. You can query these
 as a part of SQL query by appending name with the table.
 
+``$properties`` Table
```

Comment:
This comment is for Line 196. Will you be able to add some explanation there to say those properties are available for all Presto tables, not just Iceberg tables?

Reply:
@yingsu00 Under `Table Properties` in the Iceberg document some properties that are mentioned are I think specific to the Iceberg table like `partitioning`, `location` & `format_version`.
For hive connector, these are `partitioned_by `, `external_location`  respectively & `format_version` is not there for Hive.
Please correct me if you were talking about something else here.

Reply:
@agrawalreetika yes I mean it would help the users to mention if they are Iceberg specific, or the same with Hive, just as you explained above.